spark/src/test/resources/tpcds-micro-benchmarks/join_exploding_output.sql

```diff
@@ -17,6 +17,6 @@
 
 -- This is based on the first join in q72 when there is no join reordering
 
-select count(*)
+select cs_order_number, cs_quantity, inv_quantity_on_hand
```

Comment:
I changed these queries so that they are not aggregate queries

Reply:
What the motivation behind this change?

Reply:
Just to make the test focus on join rather than join + aggregate. We have separate tests already for different types of aggregate.